docs/api-testing/use-cases/saving-token-kv.md

```diff
@@ -0,0 +1,278 @@
+---
+id: saving-token-kv
+title: Saving a Token in a Key/Value Store
+sidebar_label: Saving Token in K/V Store
+description: Saving a Token in a Key/Value Store
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+One of the common scenarios you can face when you are working with APIs is authentication. Most of the time you call the endpoint that lets you authenticate and then use the token in the following calls. Sometimes, there may be cases where you can't call the endpoint every time, hence you need to save the token in a variable and use it more times. If you have only one test, you can reuse the same token easily. What happens if you need it in more tests inside your project or across the whole organization? The Vault is not the solution because it contains static values and you have to manually update the value every time. Here is when the Key/Value store can help.
```

Comment:
```suggestion
One of the common scenarios you can face when you are working with APIs is authentication. Most of the time you call the endpoint that lets you authenticate and then use the token in the following calls. Sometimes, there may be cases where you can't call the endpoint every time, hence you need to save the token in a variable and use it more times. If you have only one test, you can reuse the same token. What happens if you need it in more tests inside your project or across the whole organization? The Vault is not the solution because it contains static values and you have to manually update the value every time. Here is when the Key/Value store can help.
```